scripts/run-clang-format.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python
+
+# This is taken from: https://github.com/Sarcasm/run-clang-format
```

Comment:
License for both is MIT, but the copyright info would still be different.  I'm not sure how this should normally be handled.

Reply:
Hm, I wasn’t really paying that much attention to licensing so far TBH.
I was under the impression that as long as we don’t re-distribute or link to these things, it doesn’t really matter. It is a dev-only dependency.

Reply:
Maybe add the necessary copyright notice, then we're fine.